core/templates/components/oppia-angular-root.component.ts

```diff
@@ -550,6 +552,9 @@ export class OppiaAngularRootComponent implements AfterViewInit {
   static endExplorationRulesService: EndExplorationRulesService;
   static endExplorationValidationService: EndExplorationValidationService;
   static explorationDiffService: ExplorationDiffService;
+  static explorationCreationBackendApiService:
+    ExplorationCreationBackendApiService;
+  static explorationCreationService: ExplorationCreationService;
```

Comment:
Don't add any new lines to oppia-angular-root. Can you remove these?

Reply:
done